crates/esplora/src/lib.rs

```diff
@@ -1,22 +1,63 @@
 #![doc = include_str!("../README.md")]
 
-//! This crate is used for updating structures of [`bdk_chain`] with data from an Esplora server.
+//! # Primary Methods
 //!
-//! The two primary methods are [`EsploraExt::sync`] and [`EsploraExt::full_scan`]. In most cases
-//! [`EsploraExt::sync`] is used to sync the transaction histories of scripts that the application
-//! cares about, for example the scripts for all the receive addresses of a Wallet's keychain that it
-//! has shown a user. [`EsploraExt::full_scan`] is meant to be used when importing or restoring a
-//! keychain where the range of possibly used scripts is not known. In this case it is necessary to
-//! scan all keychain scripts until a number (the "stop gap") of unused scripts is discovered. For a
-//! sync or full scan the user receives relevant blockchain data and output updates for [`bdk_chain`]
-//! via a new [`TxGraph`] to be appended to any existing [`TxGraph`] data.
+//! The two primary methods are [`EsploraExt::sync`] and [`EsploraExt::full_scan`].
 //!
-//! Refer to [`example_esplora`] for a complete example.
+//! [`EsploraExt::sync`] is used to sync against a subset of wallet data. For example, transaction
```

Comment:
All this documentation seems redundant since we have `SynRequest` and `FullScanRequest`.

> This library is primarily intended to satisfy `SyncRequest`s with [`sync`] and [`FullScanRequest`]s with [`full_scan`]. 

To know what that means they can read the documentation for those types.

Reply:
Removed.